zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sRemoteInterpreterProcess.java

```diff
@@ -223,6 +245,13 @@ public boolean isRunning() {
     return false;
   }
 
+  public String getPodPhase() {
+    Pod pod = client.pods().inNamespace(namespace).withName(podName).get();
+    if (pod != null) {
+      return pod.getStatus().getPhase();
+    }
+    return "Unkown";
```

Comment:
typo

Reply:
Thanks for the hint, I have corrected that